packages/libsolv/libsolv.spec

```diff
@@ -283,8 +277,8 @@ mv %{buildroot}%{_bindir}/repo2solv{.sh,}
 %endif
 
 %changelog
-* Thu Oct 25 2018 Patrick Creech <pcreech@redhat.com> - 0.6.34-2.pulp
-- Build version for pulp 2.18
+* Thu Aug 09 2018 Igor Gnatenko <ignatenkobrain@fedoraproject.org> - 0.6.35-1
+- Update to 0.6.35
```

Comment:
it inserted a chronological issue with the changelog.  I figured it wasn't  major issue to just remove it, since technically we're rebasing on what's in fedora.